tv_apps/smart-system/js/app_window_factory.js

```diff
@@ -99,8 +101,64 @@
       }
     },
 
+    handlePresentation: function awf_handlePresentation(detail) {
+      var parseUrl = detail.url.split('/');
+      var protocol = parseUrl[0].toUpperCase();
+      var manifestURL;
+
+      parseUrl.length = 3;
```

Comment:
We should write some comments on why we need to reset the length of parseUrl.

It should be because that
```
app://xxx.yyy.com/aa.index.html
 =>>  'app:', '', 'xxx.yyy.com'
```

If an user types `apps:///`, the manifestURL will be `app:///manifest.webapp`. Is it fine to us?